.github/workflows/test-package.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         run: dart format --output=none --set-exit-if-changed .
         if: always() && steps.install.outcome == 'success'
       - name: Analyze code
-        run: dart analyze --fatal-infos
+        run: dart analyze
```

Comment:
Yes, but I will revert this after a few more PRs; the fixes to the analysis issues have already landed in google3, they just need to be sync'd out still (and I didn't want the analysis lints here to prevent the tests from running on this PR).

Reply:
Re-enabling in https://github.com/dart-lang/intl/pull/468.